### PR TITLE
Add Data Tables Reference

### PR DESCRIPTION
**Pull request by CommanderFoo**

### CODE REVIEW

File: src/references/data_tables.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+title: Data Tables
+tags:
+  - Reference
+---
+
+# Data Tables
+
+Creators can create **Data Tables** that contain data in tabular form that contains rows and columns which can be access from a Lua script. Think of a data table as a list where each item is also a list. For example, a player's inventory that has items, and each item has a name, description, quantity, and icon.
+
+## Overview
```
Comment (DoubleA-MG) [inline src/references/data_tables.md]: Overview is blank? Probably meant to be up on line 10

File: src/references/data_tables.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+#### Edit Column Name
+
+Creators can modify column names by double-clicking on the column name in the data table. Column names must be unique similar to how a Lua table can not have keys with the same name.
+
+![!Edit Column Name](../img/DataTables/edit_column_name.png){: .center loading="lazy" }
+
+#### Edit Column Type
+
+Creators can modify the column type by clicking on the warning icon in the column header. The **Column Settings** window will pop up allowing creators to modify the type and the column name. All columns need a type, otherwise accessing any of the columns from a Lua script will show an error in the **Event Log**.
```
Comment (DoubleA-MG) [inline src/references/data_tables.md:47]: Perhaps a warning that the type of a row cannot be changed easily.

File: src/references/data_tables.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+### Copy and Pasting Columns and Rows
+
+Creators can copy and paste existing rows and columns into the data table by right clicking and selecting **Copy** or **Paste** from the pop up option panel.
+
+![!Copy and Pasting](../img/DataTables/copy_paste.png){: .center loading="lazy" }
+
+### Editing Outside of the Core Editor
+
+A **Data Table** is a text file called a `csv` (Command-separated Values). This allow creators to easily modify the data of a data table. So adding large amounts of data would be easier by editing the csv file rather then adding the rows in one by one manually.
```
Comment (DoubleA-MG) [inline src/references/data_tables.md]: Comma-Separated Values

**Review by DoubleA-MG — CHANGES_REQUESTED** [review]
Feel like there might be need for a final h2 (Learn More or Use Cases)

File: src/references/data_tables.md (hunk 2 above)
Comment (CommanderFoo) [reply src/references/data_tables.md:47]: I did have this in, but removed it, as it was changed. But looks to be removed again. Not sure why.

**Review by DoubleA-MG — APPROVED** [review]

**Pull request merged**